smallrye-reactive-messaging-in-memory/src/main/java/io/smallrye/reactive/messaging/connectors/i18n/InMemoryMessages.java

```diff
@@ -0,0 +1,12 @@
+package io.smallrye.reactive.messaging.connectors.i18n;
+
+import org.jboss.logging.Messages;
+import org.jboss.logging.annotations.MessageBundle;
+
+@MessageBundle(projectCode = "SRRML")
+public interface InMemoryMessages {
+
+    InMemoryMessages msg = Messages.getBundle(InMemoryMessages.class);
+
+    // 19334-19666 (messaging)
```

Comment:
Should it be before the definition? Or actually, now that I'm thinking about it, in the Javadoc of the class?

Reply:
There is a global file in the root of the parent that contains all indexes. I can move this to Javadoc, no problem :)

Reply:
Moved to JavaDoc